forge-gui/res/cardsfolder/upcoming/livaan_cultist_of_tiamat.txt

```diff
@@ -0,0 +1,9 @@
+Name:Livaan, Cultist of Tiamat
+ManaCost:2 R
+Types:Legendary Creature Dragon Shaman
+PT:1/3
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ TrigPump | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a noncreature spell, target creature gets +X/+0 until end of turn, where X is that spell's mana value.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | NumAtt$ +X
+SVar:X:TriggeredCard$CardManaCost
```

Comment:
This should probably be `TriggerCount$CastSACMC`
(it matters if it's X spell that gets countered before trigger resolves)